### ISSUE TITLE
Pollution of global namespace.

### ISSUE THREAD

**Issue by beyondlimits** [minetest-mods/digilines#38]
I have found that digilines mod leaks a lot of stuff to global namespace.
```
box_bump1
box_bump2
box_center
box_xm
box_xmy
box_xp
box_xpy
box_zm
box_zmy
box_zp
box_zpy
create_lines
digiline
generate_line
generate_texture
tiles
wiredesc
```
By the way, should digilines own namespace be `digilines` rather than `digiline`?

**Comment by kaeza** [comment]
#44

**Comment by DS-Minetest** [comment]
To be closed?

**Comment by OgelGames** [comment]
This should be closed, it was all fixed in #44 and #47.

@SmallJoker

**Comment by SmallJoker** [comment]
Yes. Please @ me when something needs action. I do not have the time to check all new comments/issues.
